src/angular/src/app/spark-core-angular/components/sprk-narrow-navigation-item/sprk-narrow-navigation-item.component.ts

```diff
@@ -7,23 +7,32 @@ import * as _ from 'lodash';
     <li [ngClass]="getClasses()">
       <div *ngIf="subNav != null; then menu; else link;"></div> 
       <ng-template #link>
-        <a [attr.aria-controls]="controls_id" class="sprk-c-Accordion__summary" [routerLink]="href">
+        <a [attr.aria-controls]="controls_id" 
+           class="sprk-c-Accordion__summary" 
+           [routerLink]="href"
+           [attr.data-analytics]="analyticsString">
           <span class="sprk-b-TypeBodyTwo sprk-c-Accordion__heading ">
             {{ text }}
           </span>
         </a>
       </ng-template>
       <ng-template #menu>
-        <a [attr.aria-controls]="controls_id" class="sprk-c-Accordion__summary" href="#nogo"
-        (click)="toggleAccordion($event)">
+        <a [attr.aria-controls]="controls_id" 
+           class="sprk-c-Accordion__summary" 
+           href="#nogo"
```

Comment:
Might of forgot to hook up this href to be customizable here.

Reply:
this one is part of the template that doesnt redirect on that <a>, still needs the href for focusing though